README.md

```diff
@@ -1,12 +1,13 @@
 # Neptune + scikit-learn integration
 
-Experiment tracking, model registry, data versioning, and live model monitoring for scikit-learn (sklearn) trained models.
+Experiment tracking for scikit-learn (sklearn) trained models.
```

Comment:
I believe this would be the way to write it, if desired. In any case, `(sklearn)` can be left out.

```suggestion
Experiment tracking for scikit-learn&ndash;trained models.
```